safe/definitions/constants.py

```diff
@@ -77,3 +77,9 @@
 
 # Layer properties
 MULTI_EXPOSURE_ANALYSIS_FLAG = 'multi_exposure_analysis'
+
+
+# Smoothing mode
+NONE_SMOOTHING = 'none_smoothing'
+NUMPY_SMOOTHING = 'numpy_smoothing'
+SCIPY_SMOOTHING = 'scipy_smoothing'
```

Comment:
Quick note, just noticed this weekend that Python as a Enum function: https://docs.python.org/3/library/enum.html
It has been backported to Python 2.7.

Reply:
ah nice, we should use it in InaSAFE